recipes/easyloggingpp/all/conanfile.py

```diff
@@ -54,6 +55,10 @@ def config_options(self):
         if self.settings.os == "Windows":
             del self.options.fPIC
 
+    def validate(self):
+        if self.settings.compiler.cppstd:
```

Comment:
Conan 2 always defines a cppstd, so this check can be removed an always call `check_min_cppstd`

Reply:
I removed the check and bumped `required_conan_version` to 2.0: 9ace18fbe58144bea5a94f0a826ca1e4e77a9063